src/utils/radio.ts

```diff
@@ -0,0 +1,7 @@
+export const getRadioStationUrl = (stationName: string | boolean) => {
+  if (!stationName) {
+    return "";
+  }
+
+  return `https://w.soundcloud.com/player/?url=${stationName}&amp;start_track=0&amp;single_active=true&amp;show_artwork=false`;
+};
```

Comment:
Two ambiguities:

1. Why is boolean accepted value and what will happen when given `true`? Could a type of `string | "true"` serve better purpose?

2. Is the empty string sensible return value based on expected use? Could "#" serve better? This is just a "please provide more info question" since it should be well reasonable to have empty string or `null` just as well

Reply:
1. Removed `| boolean` type and `!stationName` check at all, moved true/false check directly to component, so we run the function only if we have proper radio station
2. Both '' and `#` should have similar  results upon load, so that doesn't really matter
433dfe8